### ISSUE TITLE
Clean Up And Refactor VR3 (an ongoing task)

### ISSUE THREAD

**Issue by tbpassin** [leo-editor/leo-editor#4026]
Removing technical debt for VR3 will be an ongoing activity, to be accomplished by a number of PRs over time.  Some bug fixes will be addressed as they are found.

**Comment by edreamleo** [comment]
I am going to close this issue. PR #4017 cleans up VR3 in several ways. Most notably, neither VR nor VR3 needs to know about layout-related issues.  Please create new issues and PRs as needed.

**Comment by tbpassin** [comment]
This PR wasn't about layouts. I'm fine if you want to do some tweaks on VR3, but **not** if I've got a PR open. This is the second time this has happened in the last couple of months.  Last time I had to go figure out what I had changed and fit that into your merged changes. Now I'm going to have to figure it out all over again.

My plan, if you were going to do many changes to devel (that didn't involve VR3) was to copy the file, merge the new devel into my clone, and then copy my changed file back into the tree.  Then I could commit it and create a PR.

Now I'll have to figure out how to merge my changes with the current state of devel.  I'm not even sure how to go about it.
